gridstatus/spp.py

```diff
@@ -707,6 +707,56 @@ def _process_operating_reserves(self, df):
 
         return df
 
+    @support_date_range("DAY_START")
+    def get_day_ahead_marginal_clearing_prices(self, date, end=None, verbose=False):
+        """Provides Marginal Clearing Price information by Reserve Zone for each
+        Day-Ahead Market solution for each Operating Day.
+        Posting is updated each day after the DA Market results are posted.
+        Available at https://portal.spp.org/pages/da-mcp#
+
+        Args:
+            date: date to get data for
+            end: end date
+            verbose: print url
+
+        Returns:
+            pd.DataFrame: Day Ahead Marginal Clearing Prices
+        """
+        if date == "latest":
+            raise ValueError(
+                "Latest not supported for Day Ahead Marginal Clearing Prices",
+            )
+
+        url = f"{FILE_BROWSER_DOWNLOAD_URL}/da-mcp?path=/{date.strftime('%Y')}/{date.strftime('%m')}/DA-MCP-{date.strftime('%Y%m%d')}0100.csv"  # noqa
+
+        msg = f"Downloading {url}"
+        log(msg, verbose)
+        df = pd.read_csv(url)
+
+        return self._process_day_ahead_marginal_clearing_prices(df)
+
+    def _process_day_ahead_marginal_clearing_prices(self, df):
```

Comment:
for consistency with other isos we do as_prices for, lets add a market = "DAM" column to this 

for example ercot: https://github.com/kmax12/gridstatus/blob/main/gridstatus/ercot.py#L1460

Reply:
Done in the `assign` below.